src/demo/demo.js

```diff
@@ -1,5 +1,6 @@
-console.log("demo.js")
 import { h, render, Component } from 'preact'
+import URLSearchParams from 'url-search-params'
+import { getInitSdkOptions } from './demoUtils'
 
```

Comment:
I think for legacy reasons we still need to keep some query parameters. `?liveness=true` is being used for demo. And I think there are other parameters which are used for testing

Reply:
All of the query params should still work - on both pages. Are there ones you're seeing that aren't?

Reply:
just tested it, https://staging-557-pr-onfido-sdk-ui-onfido.surge.sh/demo.html?liveness=true 
It is working :) ignore me